testing/endtoend/endtoend_setup_test.go

```diff
@@ -82,6 +82,10 @@ func e2eMinimal(t *testing.T, cfgo ...types.E2EConfigOpt) *testRunner {
 	for _, o := range cfgo {
 		o(testConfig)
 	}
+	//TODO: web3signer does not currently support validator registration signing so evaluator will break code
+	if !testConfig.UseWeb3RemoteSigner {
```

Comment:
You should shift this to the `evals` variable and before we set it into the config struct. 

`evals = append(evals, ev.FeeRecipientIsPresent)`

Reply:
oh wait i realized that I had this here because the useweb3remotesinger property is set through the functional options in cfgo so I don't know this type until I run the config through it.

Reply:
Ah that is unfortunate, in that case feel free to leave it as it is